core/src/main/java/nl/nn/adapterframework/core/PipeLine.java

```diff
@@ -79,11 +76,20 @@
  * If the processing of the message concluded without exceptions and the status of the transaction is
  * STATUS_ACTIVE (i.e. normal) the transaction will be committed. Otherwise it will be rolled back,
  * or marked for roll back by the calling party.
-
- * </p>
- * 
+ *
  * @author  Johan Verrips
  */
+/*
+ * Processor and keeper of a line of {@link IPipe Pipes}.
+ * <br/>
+ * Pipelines also generate statics information per Pipe and keep forwards, that are registered
```

Comment:
ook hier maak je hier zomaar commentaar van.

Reply:
Ik heb de informatie die er stond in mijn tekst opgenomen. De oude tekst is nu weg.